github/tests/Framework.py

```diff
@@ -89,26 +89,30 @@ def __init__(self, file, protocol, host, port, *args, **kwds):
     def request(self, verb, url, input, headers):
         print verb, url, input, headers,
         self.__cnx.request(verb, url, input, headers)
-        fixAuthorizationHeader(headers)
+        copied_headers = headers.copy() # fixAuthorizationHeader fixes *in place* which breaks requests
+        fixAuthorizationHeader(copied_headers)
         self.__writeLine(self.__protocol)
         self.__writeLine(verb)
         self.__writeLine(self.__host)
         self.__writeLine(self.__port)
         self.__writeLine(url)
-        self.__writeLine(str(headers))
+        self.__writeLine(str(copied_headers))
         self.__writeLine(str(input).replace('\n', '').replace('\r', ''))
 
     def getresponse(self):
         res = self.__cnx.getresponse()
 
         status = res.status
-        print "=>", status
+        print("=>", status)
```

Comment:
This works on Python 2?

Reply:
You're right, it will print a tuple. We could either replace with:
```python
print("=>".format(status))
```
which work the same in both, or just leave it as is.

Since it's not the point of this PR, let's leave it as-is. But the day you want to get rid of 2to3, this would be a first babystep :)